doc/source/user_guide/session/launching_ansys_fluent.rst

```diff
@@ -15,12 +15,19 @@ so that commands can be sent to Fluent from the Python interpreter:
 
 
 You can use the :func:`connect_to_fluent() <ansys.fluent.core.launcher.launcher.connect_to_fluent>`
-function to connect to a running Fluent session that has started the gRPC server. The gRPC
-server in Fluent can be started using the ``-sifile=<server_info_file_name>`` command line
-startup option, ``server/start-server`` text command or
-``File -> Applications -> Server -> Start...`` ribbon menu. Fluent writes out a server-info file on
-starting the gRPC server. This code connects to a running Fluent session
-using a server-info file server.txt in the working directory:
+function to connect to a running Fluent session that has already started the gRPC server. There are
+some options for starting Fluent's gRPC server:
+
+#.Start Fluent with ``<fluent_executable> -sifile=<server_info_file_name>``.
+#.Execute the ``server/start-server`` TUI command in Fluent. 
+#.Execute ``File -> Applications -> Server -> Start...`` from the FLuent GUI ribbon menu.
```

Comment:
Each point above could end ", or" instead of "."

Reply:
Right.